src/webextension/locales/en-US/widgets.ftl

```diff
@@ -1,8 +1,6 @@
 copy-to-clipboard-button = Copy
 copy-to-clipboard-copied = ✔ Copied
 
-filter-input-clear = Clear
```

Comment:
Don't remove this; just set the `.title` property to `Clear`; you can see an example of the syntax immediately after this line with `modal-root`.